.github/workflows/net-8-pipeline.yaml

```diff
@@ -49,7 +46,7 @@ jobs:
         - name: configure aws credentials
           uses: aws-actions/configure-aws-credentials@v2
           with:
-            role-to-assume: ${{ secrets.AWS_ROLE }}
+            role-to-assume: ${{ secrets.AWS_LOAD_TEST_RUNNER_ROLE }}
```

Comment:
Shouldn't this be the deploy role?

Reply:
same for the other files too

Reply:
went back and re-checked, `AWS_LOAD_TEST_RUNNER_ROLE` is the correct role for the load test jobs.  The Sam Deploy role is used in the Deployment actions.

But did realize I was missing the `permission` statements in some of the docs - I've added those.